test/models/metadata.test.ts

```diff
@@ -3,6 +3,11 @@ import { expect } from '@open-wc/testing';
 import { Metadata } from '../../src/models/metadata';
 
 describe('Metadata', () => {
+  it('properly instantiates metadata with no data', async () => {
+    const metadata = new Metadata();
+    expect(metadata.identifier).to.be.undefined;
+  });
+
```

Comment:
`identifier` is the unusual case where its getter pulls directly from the `rawMetadata` with no parser in the middle and no memoization. Do we want one more test (or at least one more assertion) for the result of accessing a _parsed_ property with no data?